make/harbor.cfg

```diff
@@ -109,22 +106,30 @@ token_expiration = 30
 #Set to "adminonly" so that only admin user can create project.
 project_creation_restriction = everyone
 
-#The follow configurations are for Harbor HA mode only
-#####################################################
-#the address of the mysql database.
+#DB configuration Section
+#the address of the mysql database. Only need to change when use external db.
```

Comment:
1) when use => when using
2) the => The , since all comments is capitalized.
3) Section => section
4) suggestion: we can using ### ... ### to start a section , instead of a single #